### ISSUE TITLE
Ansible Core 2.14 - Unsupported parameters for (ansible.legacy.command) module: warn

### ISSUE THREAD

**Issue by jseiser** [ComplianceAsCode/content#9999]
#### Description of problem:

When running the ansible generated from this repo, it will fail on ansible-core 2.14+

#### Steps to Reproduce:

1. pip install ansible==7.0.0
2. Run Ansible

#### Actual Results:

Results are coming from the redhat official repo, which uses the script [here](https://github.com/ComplianceAsCode/content/blob/master/utils/ansible_playbook_to_role.py) to generate a role from the playbooks.

```
2022-12-19T20:58:08Z:     amazon-ebs.stig8: TASK [RedHatOfficial.rhel8_stig : Read signatures in GPG key] ******************
2022-12-19T20:58:09Z:     amazon-ebs.stig8: fatal: [default]: FAILED! => {"changed": false, "msg": "Unsupported parameters for (ansible.legacy.command) module: warn. Supported parameters include: stdin_add_newline, strip_empty_ends, stdin, argv, _uses_shell, _raw_params, chdir, executable, removes, creates."}
2022-12-19T20:58:09Z:     amazon-ebs.stig8:
```

#### Expected Results:

Ansible Should Run

#### Additional Information/Debugging Steps:

An example of the generated ansible looks like this

```
- name: Read signatures in GPG key
  command: gpg --show-keys --with-fingerprint --with-colons "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
  args:
    warn: false
  changed_when: false
  register: gpg_fingerprints
  check_mode: false
  tags:
  - CCE-80795-8
  - CJIS-5.10.4.1
  - NIST-800-171-3.4.8
  - NIST-800-53-CM-5(3)
  - NIST-800-53-CM-6(a)
  - NIST-800-53-SC-12
  - NIST-800-53-SC-12(3)
  - NIST-800-53-SI-7
  - PCI-DSS-Req-6.2
  - ensure_redhat_gpgkey_installed
  - high_severity
  - medium_complexity
  - medium_disruption
  - no_reboot_needed
  - restrict_strategy
  when:
  - ensure_redhat_gpgkey_installed | bool
  - high_severity | bool
  - medium_complexity | bool
  - medium_disruption | bool
  - no_reboot_needed | bool
  - restrict_strategy | bool
```

This syntax is no longer valid in 2.14, and has been deprecated since Ansible Core 2.11, which maps to Ansible 4.  
```
  args:
    warn: false
```

* https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html
* https://docs.ansible.com/ansible-core/2.13/collections/ansible/builtin/command_module.html#parameter-warn
* https://docs.ansible.com/ansible-core/2.14/collections/ansible/builtin/command_module.html

**Comment by marcusburghardt** [comment]
Thanks for reporting this issue @jseiser .
It was recently fixed by #9807 but the content you mentioned was not yet updated.
I will keep this issue opened for now until the mentioned repository is updated.

**Comment by marcusburghardt** [comment]
RedHatOfficial.rhel8_stig is currently aligned to 0.1.68 where this issue should be resolved.
